lib/bitstyles_phoenix/bitstyles.ex

```diff
@@ -25,11 +27,24 @@ defmodule BitstylesPhoenix.Bitstyles do
   # If it does exist, then doing this renaming in the classname/2 function would make it impossible
   # for users of older bitstyles to use the "class-name-A" classname.
 
-  def classname(class, version) when version >= "5.0.0" do
+  defp downgrade_classname(class, target_version, _current_version)
+       when target_version > {5, 0, 1} do
```

Comment:
```suggestion
       when target_version > {5, 0, 0} do
```
probably there is no 5.0.1 release, right?

Reply:
There is a 5.0.1 release. Version 5.0.0 had a broken build (the CSS bundle was missing a bunch classes) of and cannot be used from the CDN that we're using, so Darren released 5.0.1 with the correct build